packages/js/src/dashboard/index.js

```diff
@@ -83,8 +83,11 @@ export { Dashboard } from "./components/dashboard";
  */
 
 /**
- * @typedef {Object} WidgetTypeInfo The widget info. Should hold what the UI needs to let the user pick a widget.
- * @property {WidgetType} type The widget type.
+ * @typedef {Object} WidgetTypes The widget info. Should hold what the UI needs to let the user pick a widget.
```

Comment:
I feel this is redundant with the `WidgetType` already there. I don't think there is a need to ever pass this type around.

But if you really want to have these two types together like this, the description could be improved?
```suggestion
 * @typedef {Object} WidgetTypes All the WidgetTypes in an object, preventing hardcoded strings around the codebase.
```